src/pybullet_helpers/geometry.py

```diff
@@ -0,0 +1,78 @@
+"""Pybullet helper class for geometry utilities."""
+from __future__ import annotations
+
+from typing import NamedTuple, Sequence, Tuple
+
+import numpy as np
+import pybullet as p
+from pybullet_utils.transformations import euler_from_quaternion, \
+    quaternion_from_euler
+
+from predicators.src.structs import Array
+
+Pose3D = Tuple[float, float, float]
+Quaternion = Tuple[float, float, float, float]
+RollPitchYaw = Tuple[float, float, float]
+
+
+class Pose(NamedTuple):
+    """Pose which is a position (translation) and rotation.
+
+    We use a NamedTuple as it supports retrieving by integer indexing
+    and most closely follows the pybullet API.
+    """
+
+    position: Pose3D
+    quat_xyzw: Quaternion = (0.0, 0.0, 0.0, 1.0)
```

Comment:
thoughts about just making this attribute called `orientation` and removing the orientation property?

Reply:
I think that makes sense since it's what pybullet returns as well.

It may only get confusing when we use other simulators as well which use rpy, or wxyz quaternion representation. But let's worry about that when it happens to keep things cleaner pybullet side.